Basic.wl

```diff
@@ -2,13 +2,13 @@
 
 BeginPackage["CATemplates`Basic`"];
 
-
+BooleanFromRule::usage="Transforms a rule into boolean form.";
+ConjunctionFromRuleSet::usage="Transforms a rule set into a boolean conjunction";
+DNFFromRuleSet::usage="Transforms a set of set of rules into a DNF form boolean expression";
+MinimizedRuleSets::usage = "Minimize a subset boolean expression";
```

Comment:
Avoid adding functions to `Basic`. 
catemplates#59 removes this namespace and we are gonna have conflicts.
If these functions are only used on `ExceptionTemplates`, you can leave them there.